tests/cli/workflow/balsamic/conftest.py

```diff
@@ -523,21 +519,20 @@ def balsamic_store(base_store: Store, balsamic_dir: Path, helpers) -> Store:
         _store,
         internal_id="balsamic_case_mixed_paired_error",
         family_id="balsamic_case_mixed_paired_error",
+        data_analysis="balsamic",
```

Comment:
This fixture will no longer be needed since it checks whether data analysis is different on sample level

Reply:
I cant, then the test test_error_mixed_application fails then.. Maybe I'm removing another fixture then the one you are thinking about?

Reply:
:+1:  nvm my bad